docs/api-docs/partner/staying-current.md

```diff
@@ -0,0 +1,45 @@
+
+For a complete list of available resources, see [Support](https://developer.bigcommerce.com/api-docs/partner/getting-started/support). 
+
+## Cornerstone theme updates
+
+For recent updates to the Stencil's Cornerstone theme, we recommend setting up [a feed alert](https://developer.bigcommerce.com/changelog#feeds) for the [Developer Changelog](https://developer.bigcommerce.com/changelog). You can also refer to the [Cornerstone repository](https://github.com/bigcommerce/cornerstone/) in GitHub.
+
+In most cases, your theme will update automatically. Occasionally, you may have to take some action by opting in to the new feature in the control panel.
+
+### Custom themes
+
+Depending on the level of customization, you may need to manually port changes into your custom theme. The [Theme Updates and Version Control](https://developer.bigcommerce.com/stencil-docs/deploying-a-theme/theme-updates-and-version-control) article talks about the steps you can take to keep your custom theme up-to-date with BigCommerce theme updates and version releases. To learn more about customization and applying updates, see [Apply Updates](https://developer.bigcommerce.com/stencil-docs/developing-further/updates).
+
+## API updates
+
+To keep up with API changes, we recommend setting up [a feed alert](https://developer.bigcommerce.com/changelog#feeds) for the [Developer Changelog](https://developer.bigcommerce.com/changelog).
+
+As BigCommerce continues to deprecate V2 API endpoints and operations, consider migrating your store to the V3 API to take advantage of new features and improved performance. For more information on V3 improvements and to learn about the differences between the V2 and V3 APIs, see [Difference between V2 and V3 Catalog REST APIs](https://developer.bigcommerce.com/api-docs/store-management/catalog/v2-vs-v3). To compare major V2 and V3 operations, see [V2 to V3 Catalog Operations Comparison](https://developer.bigcommerce.com/api-docs/store-management/catalog/v2-v3-examples). For planned deprecations and sunsets, refer to [Deprecations and Sunsets](https://developer.bigcommerce.com/api-docs/getting-started/deprecations-and-sunsets).
+
+### Open betas
+
+Our [Product Blog](https://support.bigcommerce.com/s/blog) is the best place to learn about the upcoming open betas. 
+
+## Checkout SDK updates
+
+The Checkout JavaScript SDK and associated documentation are available from the [Checkout SDK repository](https://github.com/bigcommerce/checkout-sdk-js). For information on [Optimized One-Page Checkout](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout), see the [Checkout JS repository](https://github.com/bigcommerce/checkout-js).
```

Comment:
@ektravel, a bit confusing having two links to click for information on Optimized One-Page Checkout. Best remove the link to support.